micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/configs/UICustomizations.js

```
@@ -0,0 +1,272 @@
+          },
+        ],
+      };
+      data.config.select = (data) => {
+        const temp = data?.MdmsRes?.["sandbox-ui"]?.ModuleMasterConfig?.find((item) => item?.module === additionalDetails?.moduleName);
+        return {
+          module: temp?.module,
+          master: temp?.master?.filter((item) => item.type === "module" || item.type === "common" || item.type === "boundary"),
+        };
+      };
+
+      delete data.body.custom;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)